### ISSUE TITLE
Add support for more shapes in FilterResponseNormalization

### ISSUE THREAD

**Issue by Squadrick** [tensorflow/addons#1441]
#765 added FRN to TFA. 

Currently, `tfa.layers.FilterResponseNormalization` only support 4D tensors of order `NHWC`. We need to add support for arbitrary tensors and let the user specify the axis/axes to normalize over. (axes currently is forced to be `[1, 2]`). 

See the TODO in `tensorflow_addons/layers/normalizations.py`.

BLOCKED: Till #765 is merged.

**Comment by Opletts** [comment]
@Squadrick I'll take this up.

**Comment by Suraj-Upadhyay** [comment]
Hii Guys, I am working on this. Currently reading the research paper for [Filter Response Normalization](https://arxiv.org/pdf/1911.09737.pdf).

If in any doubts I will post it in the comments here and I hope to send a PR soon.

**Comment by Opletts** [comment]
Sorry guys, I completely forgot about this issue! I have #1496 open and I'll finish it this week.

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
